src/main/java/net/snowflake/ingest/connection/ServiceResponseHandler.java

```diff
@@ -205,6 +256,21 @@ private static void handleExceptionalStatus(
               apiName,
               requestId == null ? "" : requestId.toString());
           throw new BackOffException();
+        case HttpStatus.SC_UNAUTHORIZED:
+          LOGGER.warn(
+              "401 Status hit from {}, requestId:{}",
+              apiName,
+              requestId == null ? "" : requestId.toString());
```

Comment:
Duplicate logic, could you log it before the all cases?

Reply:
Fixed.